data-pipeline/device-profile-updater/src/test/java/org/ekstep/ep/samza/service/DeviceProfileServiceTest.java

```diff
@@ -127,7 +169,8 @@ public void shouldNotUpdateFirstAccessifPresent() throws Exception {
     }
 
     @Test
-    public void shouldUpdateFirstAccessifNotPresent() {
+    public void shouldUpdateFirstAccessifNotPresent() throws Exception{
```

Comment:
Add similar test cases to the updated on field

Reply:
Done.